### PR TITLE
Add descriptions 

### PR DESCRIPTION
**Pull request by froydis98**
There will be a new PR with the associated styling for the descriptions

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/QuestionsAndAnswersDisplay/QuestionsAndAnswersDisplay.tsx [hunk 1]
```diff
@@ -73,8 +73,8 @@ const QuestionsAndAnswersDisplay = (
                             : props.questionAndAnswer.answers?.id
                     }
                 >
-                    {item.valueBoolean && <p>Sant</p>}
-                    {item.valueBoolean === false && <p>Usant</p>}
+                    {item.valueBoolean && <span>Sant</span>}
+                    {item.valueBoolean === false && <span>Usant</span>}
```
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/QuestionsAndAnswersDisplay/QuestionsAndAnswersDisplay.tsx:77]: Any reason these elements are spans and not p, considering the content is just text?

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx [hunk 2]
```diff
@@ -98,32 +196,59 @@ const SchemaView = (props: SchemaViewProps) => {
...
-                                                                                .answers
-                                                                                ?.answers
-                                                                                .answer &&
-                                                                            qa
-                                                                                .answers
-                                                                                ?.answers
-                                                                                .answer[0] && (
+                                                                                2 && (
                                                                                 <div
-                                                                                    className="qa-container"
```
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx:111]: Why was the classname for this div removed?

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx [hunk 3]
```diff
@@ -84,12 +84,110 @@ const SchemaView = (props: SchemaViewProps) => {
...
+                                                                )}
+                                                                <Col
+                                                                    span={6}
+                                                                    order={2}
+                                                                    className="section-description"
+                                                                >
+                                                                    {!sectionDescription.includes(
+                                                                        section.id.split(
+                                                                            '.',
+                                                                        )[0],
```
Comment (erlinvan) [inline src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx:127]: I don't think you need to split here, sine section.id wont contain '.'. Replacing this with `section.id` should be the same.

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx [hunk 4]
```diff
@@ -84,12 +84,110 @@ const SchemaView = (props: SchemaViewProps) => {
...
+                                                                            value="small"
+                                                                            onClick={() =>
+                                                                                setSectionDescription(
+                                                                                    (
+                                                                                        sectionDescription,
+                                                                                    ) => [
+                                                                                        ...sectionDescription,
+                                                                                        section.id.split(
+                                                                                            '.',
+                                                                                        )[0],
```
Comment (erlinvan) [inline src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx:140]: Same here, as the comment above

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx [hunk 5]
```diff
@@ -84,12 +84,110 @@ const SchemaView = (props: SchemaViewProps) => {
...
+                                                                            onClick={() =>
+                                                                                setSectionDescription(
+                                                                                    sectionDescription.filter(
+                                                                                        (
+                                                                                            list,
+                                                                                        ) =>
+                                                                                            list.replace(
+                                                                                                section.id.split(
+                                                                                                    '.',
+                                                                                                )[0],
```
Comment (erlinvan) [inline src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx:167]: And here.

File: src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/QuestionsAndAnswersDisplay/QuestionsAndAnswersDisplay.tsx (hunk 1 above)
Comment (froydis98) [reply src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/QuestionsAndAnswersDisplay/QuestionsAndAnswersDisplay.tsx]: Changed to div and p

**Pull request merged**